scripts/validate

```diff
@@ -15,4 +15,4 @@ if [[ -z "$(command -v golangci-lint)" ]]; then
 fi
 
 echo "Running: golangci-lint run"
-golangci-lint run
+golangci-lint run --timeout 5m
```

Comment:
I might miss something, but why we need this timeout?

Reply:
@weihanglo the default timeout is 1m. With the additional code it was taking more than 1 minute, hence the need to bump it up.